### PR TITLE
Refine for mvp

### PR DESCRIPTION
**Pull request by hcorson-dosch**
This PR gets us closer to the MVP

- It adds definitions for all pools and fluxes - #12 - will need to be updated once definitions are finalized
- It adds a caption explaining log vs linear scale - #8
- It slows down the log-linear transition - #8
- It adds a text prompt that additional information is available if you click on each row - #11 
- It adds hover interaction on on a laptop that helps the user visually see each row - #11 
- It adds a toggle to turn on or off uncertainty ranges for pools and fluxes - #9
- It adds uncertainty ranges to cards - #9
- It adds a header to the dialog card, starting to work on #14 
- It also re-styles the chart based on input from the viz team

To Do:
* Fix x-axis log scale labels so that don't use scientific notation
* Dynamically color header of dialog card
* Switch uncertainty buttons to toggle
* Update mobile view

### CODE REVIEW

File: src/components/pool_flux_chart.vue [hunk 1]
```diff
@@ -105,13 +139,13 @@ export default {
...
           // set up radio button interaction
-          this.chartContainer.selectAll("input").on("click", this.changeXScale.bind(this));
+          this.captionContainer.selectAll("input").on("click", this.changeXScale.bind(this));
           
         },
         adaptScales(data, xMin) {
           Object.keys(this.scales).forEach(function (scaleType) {
               this.scales[scaleType]
-                  .domain([xMin, this.d3.max(data, d => d.value_km_3)])
+                  .domain([xMin, this.d3.max(data, d => d.range_high)])
```
Comment (ceenell) [inline src/components/pool_flux_chart.vue:148]: The axis labels for linear are overlapping for me
image

Comment (ceenell) [reply src/components/pool_flux_chart.vue:148]: We should use labels for either millions or billions, e.g. 100 M or 0.1 B

**Review by ceenell — APPROVED** [review]
This looks good. If it takes more than a few minutes, let's work on the scale numbers in the next PR. My PR is going to conflict this one because I changed the data source within the pipeline. I'll switch it for now so that everything reads from `public/` while aws credentials are getting sorted out and you can still edit. I'll have to run the pipeline to put the images up for now

**Comment by ceenell** [comment]
The font color on the dialog box looks a little bit light

**Comment by hcorson-dosch** [comment]
> The font color on the dialog box looks a little bit light

Agreed -- editing that now

**Comment by hcorson-dosch** [comment]
I'll work on the scale numbers for the next PR

File: src/components/pool_flux_chart.vue (hunk 1 above)
Comment (hcorson-dosch) [reply src/components/pool_flux_chart.vue:148]: Yeah - I should tweak those, too.

**Pull request merged**